### DISCUSSION TITLE
路由匹配问题

### DISCUSSION THREAD

**Discussion by tb51cx** [apache/apisix#6697] [Q&A]
我想把abc.com/abc/*的内容路由到abc服务80端口号的/def/*，请问类似功能可以实现吗？也就是，网址的后缀跟服务的后缀不一样。

**Comment by guoqqqi** [comment]
Do you mean you want to access `abc.com/abc/` to get data from `abc.com:80/def`?
1. Create an upstream
```
host: abc.com
port: 80
weight: 1
````
2. Create a route with path set to `/abc`, and update this config
![image](https://user-images.githubusercontent.com/72343596/160075722-68cd3fae-1bdc-4a2f-badd-3fb648165270.png)
you can refer to this plugin https://apisix.apache.org/docs/apisix/plugins/proxy-rewrite/
3. Select this upstream for this route

**Reply by tb51cx** [reply]
非常感谢。
